ui/src/datasource/DatasourceOverview.vue

```diff
@@ -47,60 +40,74 @@
         />
 
         <template #[`item.trigger.interval`]="{ item }">
-          {{ getHoursFromMS(item.trigger.interval) }}h:{{ getMinutesFromMS(item.trigger.interval) }}m
+          {{ getFormattedInterval(item.trigger.interval) }}
+        </template>
+
+        <template #[`item.trigger.firstExecution`]="{ item }">
+          {{ getISODateString(item.trigger.firstExecution) }} {{ getISOTimeString(item.trigger.firstExecution) }}
+          <v-icon
+            v-if="isPastDate(item.trigger.firstExecution)"
+            small
+          >
+            mdi-check
+          </v-icon>
+          <v-icon
+            v-else
+            small
+          >
+            mdi-clock-outline
```

Comment:
Nice idea, but probably not worth to add that little icon.

I wonder which information here is relevant to the user... the first execution time like configured, or the last execution, or even the next planned execution? Probably doing more than one consumes too much space...

What do you think about showing the last execution date instead of the first one, and add this very icon here to the periodic column with a tooltip showing the next execution time?

Reply:
We might also think about adding a `details` component in the future that shows all configuration information, including dependent pipelines (but definitely not part of this PR)

Reply:
Showing `lastExecutionTime` and `nextExecutionTime` makes sense. We just have to be aware that both dates might not exist (yet). Thats why i choose `firstExecution` because both periodic and non-periodic datasources have that property.

> We might also think about adding a `details` component in the future that shows all configuration information, including dependent pipelines (but definitely not part of this PR)

Maybe we should even consider to split the table up into periodic and non-periodic datasources. That way, we can avoid all of this conditional logic inside the template and avoid ugly white columns.

Something else i noticed, showing the interval when the datasource isn't fetched periodicly doesn't make sense right? Currently it shows either way.

Reply:
What do you think of this for now?

![chrome_7hriInbXNq](https://user-images.githubusercontent.com/29175032/101923438-af490780-3bcf-11eb-83ec-35befbc3e574.png)

Reply:
I think that is a good pragmatic solution 👍